### PR TITLE
Add Job options to schedule method

### PR DESCRIPTION
**Pull request by aleksraiden**

### CODE REVIEW

File: src/Queue/Queue.php [hunk 1]
```diff
@@ -73,9 +73,10 @@ public function setMarshaler(MarshalerInterface $marshaler)
      * @param JobInterface $job Job
      * @param DateTime $when Date & time on when job should be ready for processing
      * @return JobInterface Job pushed
+     * @param array $options ADDJOB options sent to the client
```
Comment (Revisor) [inline src/Queue/Queue.php]: Lines 75 and 76 should be swapped

**Comment by Revisor** [comment]
One small comment in the code, otherwise good job.
Could you add a test that the ADDJOB options are correctly conveyed from `schedule()`? What do you think?

**Comment by aleksraiden** [comment]
Okey, of couse

**Comment by mariano** [comment]
@aleksraiden: could you add some test for this?

**Comment by mariano** [comment]
Implemented in 33dcf6f

**Comment by Revisor** [comment]
Good job.

**Pull request closed without merging**